docs/modules/concepts/pages/mail/mime/general.adoc

```diff
@@ -0,0 +1,33 @@
+= MIME General Description
+:navtitle: General
+
+What people casually call "email" usually refers more specifically to a
+text message written in a specified format called https://en.wikipedia.org/wiki/MIME["MIME"].
+
+(NOTE: I just made up the crap below, so please comment liberally!!)
+
+MIME is short for "Multipurpose Internet Mail Extensions". It specifies how to construct
+a message, and deals with character encodings, various file formats, and many other
+odds and ends.
+
+After the first MIME specification was published in 1982 there was no looking back. MIME
+a.k.a. "email" took the world by storm. Today it is arguably the most prevalent means
+of communicating with a distant party.
+
+Often, postal mail (i.e. a letter) is used as an analogy for email. You write a message
+on a piece of paper, put the paper in an envelope, and drop the envelope into a letterbox
+to be picked up by the post office. There are indeed some similarities with postal mail, 
+but there are also some important
+differences. For instance, when you write a letter, you are not required to annotate it
+with meta information. You could include a date, your address, and other information, and
+indeed, it is customary to do so. However, your letter would not be rejected if you 
+failed to include this this type of descriptive information. It would also be strange 
+not to include a message when writing a letter.
+A MIME message is almost the opposite. It *must* include a number of valid headers,
```

Comment:
are you sure? which headers are mandatory?

Reply:
Not sure at all. I will check. :-)

Reply:
I think this is now outdated, so I will resolve the comment.